libs/maps/include/mrpt/maps/CPointsMap.h

```diff
@@ -1090,7 +1091,7 @@ class CPointsMap : public CMetricMap,
 
    protected:
 	/** The point coordinates */
-	std::vector<float> m_x, m_y, m_z;
+	mrpt::aligned_std_vector<float> m_x, m_y, m_z;
```

Comment:
Oh! Why I didn't thought of this?
Perhaps it wasn't needed to make all containers aligned (weight, color,...) but at least X,Y,Z will be useful **for sure**.  :+1:

Reply:
Yeah. Might have been overkill for color. Sometimes color has useful information for processing and the algorithims are worth accelerating.